### PR TITLE
[CI:DOCS] Fix path to example files in contrib/systemd/README.md

### PR DESCRIPTION
**Pull request by lstolcman**
Fixes links to example files. Currently, only one link works, and the rest leads to 404.

#### Does this PR introduce a user-facing change?

```release-note
Fix path to example files in contrib/systemd/README.md
```

### CODE REVIEW

File: contrib/systemd/README.md [hunk 1]
```diff
@@ -26,7 +29,9 @@ You can refer to [this example](https://github.com/containers/podman/blob/main/c
...
 
 ### podman.service
-You can refer to [this example](https://github.com/containers/podman/blob/main/contrib/systemd/user/podman.service) for a rootless podman.service file.
+You can refer to [this example](https://github.com/containers/podman/blob/main/contrib/systemd/system/podman.service.in) for a rootless podman.service file.
+
+Note: replace `@@PODMAN@@` with path to podman binary, such as `/usr/local/bin/podman`
 
 ### podman.socket
-You can refer to [this example](https://github.com/containers/podman/blob/main/contrib/systemd/user/podman.socket) for a rootless podman.socket file.
+You can refer to [this example](https://github.com/containers/podman/blob/main/contrib/systemd/system/podman.socket) for a rootless podman.socket file.
```
Comment (Luap99) [inline contrib/systemd/README.md:37]: Given they are duplicated from the above I feel like we should drop this entirely

**Comment by lstolcman** [comment]
removed duplicated content
@Luap99 @baude please review

**Comment by rhatdan** [comment]
/lgtm
Thanks @lstolcman

**Pull request merged**